letsencrypt-apache/letsencrypt_apache/configurator.py

```diff
@@ -1308,7 +1342,8 @@ def get_version(self):
 
         """
         try:
-            stdout, _ = le_util.run_script([self.conf("ctl"), "-v"])
+            stdout, _ = le_util.run_script(
+                constants.os_constant("version_cmd").split(" "))
```

Comment:
Actually these constants for commands aren't available as command line parameters to users. So we're pretty much free to write the python lists in the constants file. And you are right, this isn't the right place to parse the string or generate a list out of it. I'll move the lists to constants directly.